lib/ecco/row_event_listener.rb

```diff
@@ -5,8 +5,11 @@ module Ecco
   class RowEventListener < EventListener
     ROW_EVENTS = [
       EventType::WRITE_ROWS,
+      EventType::EXT_WRITE_ROWS,
       EventType::UPDATE_ROWS,
+      EventType::EXT_UPDATE_ROWS,
       EventType::DELETE_ROWS,
+      EventType::EXT_DELETE_ROWS,
```

Comment:
> I'll probably refactor this and rely on https://github.com/shyiko/mysql-binlog-connector-java/blob/05bc0952c5de64fc836fcb9ca36f4b4570d21a72/src/main/java/com/github/shyiko/mysql/binlog/event/EventType.java#L191-L207 and add our own types that don't vary depending on the MySQL config and/or version.

There are some issues with JRuby and Java type, some of the issues are fixed by using `EventType.java_send(:isWrite, [EventType], type)`. Might be easier to keep the constants, but ensure we send the same type-strings as before (otherwise Bruce will break).